code/game/machinery/engi_points_vendor.dm

```diff
@@ -0,0 +1,815 @@
+	icon_state = "store"
+	verb_say = "states"
+	density = TRUE
+	anchored = TRUE
+	req_access = list(access_engine)
+	var/restricted_access = FALSE
+	var/obj/item/device/radio/radio
+	var/GBP = 0
+	var/GBPearned = 0
+	var/power_export_bonus = 0
+	var/air_alarm_bonus = 0
+	var/power_alarm_bonus = 0
+	var/fire_alarm_bonus = 0
+	var/alarm_rating = ""
+	var/prior_bonus = 2500
+	var/total_bonus = 0
+	var/GBP_alarm_cooldown = 4500
+	var/static/list/prize_list = list(
+		new /datum/GBP_equipment("Tendie",				/obj/item/weapon/reagent_containers/food/snacks/nugget,				50,		1),
+		new /datum/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,						50,		1),
+		new /datum/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,												50,		1),
+		new /datum/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,									250,	1),
+		new /datum/GBP_equipment("Advanced Indoor Fulton Pack",			/obj/item/weapon/extraction_pack/advanced,			300,	1),
+		new /datum/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,					400,	1),
+		new /datum/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,								500,	1),
+		new /datum/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,								500,	1),
+		new /datum/GBP_equipment("50 cardboard sheets",			/obj/item/stack/sheet/cardboard/fifty,						500,	1),
+		new /datum/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,									600,	1),
+		new /datum/GBP_equipment("Hardsuit x3",			/obj/item/clothing/suit/space/hardsuit,								750,	3),
+		new /datum/GBP_equipment("Jetpack Upgrade x3",		/obj/item/weapon/tank/jetpack/suit,								1000,	3),
+		new /datum/GBP_equipment("Forcefield Projector x3",		/obj/item/device/forcefield,								1500,	3),
+		new /datum/GBP_equipment("Powertools x4",			/obj/item/weapon/storage/belt/utility/chief/full,				2000,	4),
+		new /datum/GBP_equipment("Freon Canister",			/obj/machinery/portable_atmospherics/canister/freon,			2500,	1),
+		new /datum/GBP_equipment("Prototype Canister",		/obj/machinery/portable_atmospherics/canister/proto/default,	2500,	1),
+		new /datum/GBP_equipment("Advanced Magboot x3",			/obj/item/clothing/shoes/magboots/advance,					3000,	3),
+		new /datum/GBP_equipment("Reflector Box x3",			/obj/structure/reflector/box,								3500,	3),
+		new /datum/GBP_equipment("Radiation Collector x3",			/obj/machinery/power/rad_collector,						4000,	3),
+		new /datum/GBP_equipment("Radiant Dance Machine",		/obj/item/clothing/suit/space/hardsuit/ert/engi,			5000,	1),
+		new /datum/GBP_equipment("ERT Hardsuit x5",		/obj/item/clothing/suit/space/hardsuit/ert/engi,					7500,	5),
+		new /datum/GBP_equipment("Ranged RCD x4",			/obj/item/weapon/rcd/arcd,										9000,	4),
+		new /datum/GBP_equipment("Prototype Atmos Vehicle x2",			/obj/vehicle/space/speedbike/atmos,					10000,	2),
+		new /datum/GBP_equipment("Reactive Decoy Armor x5",		/obj/item/clothing/suit/armor/reactive/stealth,				11000,	5),

```

Comment:
Chronosuit was attempted to be balanced at one point. This never was.
Ok, the decoy flees. It's still invisibility on hit. Still meant to be admin only. Isn't even spawned usually because it bs.
As for never seeing it, I HAVE seen it. Kevinz and I tested it shortly after I was added. I gave it to him as RD for a round. I know how op it is.